src/olympia/devhub/templates/devhub/personas/edit.html

```diff
@@ -70,230 +58,9 @@ <h3 class="author">
-                Upon upload and form submission, the AMO Team will review your
-                updated design. Your current design will still be public in the
-                meantime.
-              {% endtrans %}
-            </p>
-
-            <div class="theme-design-form">
-              {% with required = False %}
-                {% include "devhub/personas/includes/theme_design.html" %}
-              {% endwith %}
-            </div>
-
-            {% set rereview = persona.rereviewqueuetheme_set.all() %}
-            {% if rereview.exists() %}
-              <h3>{{ _('Your previously resubmitted design, '
-                       'which is under pending review.') }}</h3>
-              <ul class="rereview-preview">
-                <li id="persona-header" class="row">
-                  <h3>{{ _('Pending Header') }}</h3>
-                  <img class="preview loaded" src="{{ rereview[0].header_url }}" alt="">
-                </li>
-              </ul>
-            {% endif %}
-          </div>
-        {% else %}
-          <span class="reupload">
-            {{ _('You may update your theme design here once it has been approved.') }}</span>
-        {% endif %}
-
-        <ul class="theme-preview">
-          <li id="persona-header" class="row">
-            <h3>{{ _('Header') }}</h3>
-            <img class="preview loaded" src="{{ persona.header_url }}" alt="">
-          </li>
-        </ul>
-        <h3 class="colors">{{ _('Select colors for your Theme.') }}</h3>
-        <ul class="colors">
-          {{ pretty_field(form.textcolor, label=_('Foreground Text'),
-                          tooltip=_('This is the color of the tab text'),
-                          validate=True) }}
-          {{ pretty_field(form.accentcolor, label=_('Background'),
-                          tooltip=_('This is the color of the tabs'),
-                          validate=True) }}
-        </ul>
-      </fieldset>
-      <fieldset id="persona-preview">
-        <legend>{{ _('Preview') }}</legend>
-        <div class="persona persona-large persona-preview">
-          <div class="persona-viewer" data-browsertheme="{{ persona.json_data }}"
-               style="background-image:url({{ persona.header_url }})">
-            <div class="details">
-              <span class="title" id="persona-preview-name">
-                {{ _("Your Theme's Name") }}</span>
-              <span class="author">
-                {% trans user_name=user.name,
-                         profile_url=url('users.profile', user.id) %}
-                  by <a href="{{ profile_url }}">{{ user_name }}</a>
-                {% endtrans %}
-              </span>
-            </div>
-          </div>
-        </div>
-      </fieldset>
-      <div class="listing-footer">
-        <button class="button prominent" disabled type="submit">{{ _('Save Changes') }}</button>
-      </div>
-    </form>
+		<p>
+			  <h1 class="error">Lightweight Theme editing has been disabled.</h1>
```

Comment:
Untranslated string

Reply:
It's not going to get translated before the tag anyway, but sure.

Reply:
If it somehow stays in for longer than anticipated it's best to be prepared :)